src/main/webapp/scripts/appscript.js

```diff
@@ -26,14 +26,20 @@ async function populateCustomerList() {
   console.log(customerJson);
 
   // Loops over this array to make the options for the Customer ID dropdown.
+
+
+  // Commented out for now because it was causing an error.
+
+  /** 
   var vendorHtml = '';
   for (var i = 0; i < customerJson.length; i++) {
-    html += '<option value="' + customerJson[i] + '">'
+    vendorHtml += '<option value="' + customerJson[i] + '">'
     + customerJson[i] + '</option>';
   }
 
   // Adds the options to the page, allowing them to be viewed. 
   document.getElementById('customer-ids').innerHTML = vendorHtml;
+  */
```

Comment:
Can you remove the commented out code from the PR?

Reply:
Sure, but this will be fixed once Vncent's PR is merged.

Reply:
+1 
Dropdown functionality is fixed in my PR